docs/runbooks/CDIOperatorDown.md

```diff
@@ -0,0 +1,31 @@
+# CDIOperatorDown
+
+## Meaning
+
+The CDI Operator is responsible for deploying and managing the CDI infrastructure components, such as the datavolume/PVC controllers.  
```

Comment:
@akalenyu Please add this description of what CDI is to the other alert runbooks.

Reply:
Done